packages/cli/src/tests/synthql.config.json

```diff
@@ -16,6 +16,22 @@
             "last_name": {
                 "includable": { "type": "boolean", "const": false }
             }
+        },
+        "public.niggas": {
```

Comment:
I don't know jim... I feel like we shouldn't use this word in an open source repo 😅

Reply:
I agree.. removed already